conans/client/toolchain/visual.py

```diff
@@ -0,0 +1,75 @@
+import os
+
+from conans.client.tools.win import vs_installation_path
```

Comment:
I believe we are moving the needed tools to isolate them from *utils*. Why not this `vs_installation_path`?

Reply:
This one I didn't need any modification yet. The things that I have moved have been already modified, typically simplified, removed env-vars checking, etc. I can see moving a new copy of ``vs_installation_path`` here soon, just I didn't want to add more things to this PR, just a example of the new internal utils (not exposed as tools to users, unless 200% sure, and probably way later down the road, once the toolchains are matured)